Sources/WebSocketKit/WebSocketHandler.swift

```diff
@@ -14,17 +14,30 @@ extension WebSocket {
         /// Maximum frame size after aggregation.
         /// See `NIOWebSocketFrameAggregator` for details.
         public var maxAccumulatedFrameSize: Int
-
+        
+        /// Enables PMCE if present. Defaults to 'nil'
+        /// See`PMCE` for details.
+        public var deflateConfig: PMCE.DeflateConfig? = nil
+        
+        public init(withDeflateConfig deflateConfig:PMCE.DeflateConfig) {
+            self.minNonFinalFragmentSize = 0
+            self.maxAccumulatedFrameCount = Int.max
```

Comment:
Again, we don't want to implicitly set frame sizes to accumulate completely. This opens us up to denial of service aggacks